### PR TITLE
feat(#2192): add download button in asset management overview

### LINKED ISSUE #2192: Add download button for assets in StreamPipes
### Body

StreamPipes allows the creation so called assets. All assets can be viewed in the module `Assets`.

image

Unfortunately, this view currently does not allow to download an asset directly.
Therefore we should add a download action directly beneath the remove action for every file.

Similar to #2074 

### Mentoring

As this issue is marked as a good first issue: one of @dominikriemer, @tenthe, or @bossenti are happy to provide help for getting started, just tag (one of) them if you want to start working on this issue and need some help.
This issue is ideal for people who are interested in getting started with StreamPipes and what to explore both worlds: the backend and the UI.

### StreamPipes Committer

I acknowledge that I am a maintainer/committer of the Apache StreamPipes project.

### PR DESCRIPTION
**Pull request by IsaakKrut**
Purpose
Closes #2192 

Remarks
PR introduces (a) breaking change(s): no

PR introduces (a) deprecation(s): no

### CODE REVIEW

File: ui/src/app/assets/components/asset-overview/asset-overview.component.ts [hunk 1]
```diff
@@ -147,4 +150,26 @@ export class SpAssetOverviewComponent implements OnInit {
...
+    downloadAsset(asset: SpAssetModel) {
+        this.dataExportService
+            .getExportPreview([asset._id])
+            .pipe(
+                mergeMap(preview =>
+                    this.dataExportService.triggerExport(preview),
+                ),
+            )
+            .subscribe((data: Blob) => {
+                const blob = new Blob([data], { type: 'application/zip' });
```
Comment (tenthe) [inline ui/src/app/assets/components/asset-overview/asset-overview.component.ts:163]: Hi @IsaakKrut,
Is it also possible to use the file-saver API here? If not, the existing approach is ok for me.

File: ui/src/app/assets/components/asset-overview/asset-overview.component.ts [hunk 2]
```diff
@@ -147,4 +150,26 @@ export class SpAssetOverviewComponent implements OnInit {
...
+                const url = window.URL.createObjectURL(blob);
+                const anchor = document.createElement('a');
+
+                anchor.href = url;
+                anchor.download = 'data_export';
```
Comment (bossenti) [inline ui/src/app/assets/components/asset-overview/asset-overview.component.ts:168]:
```suggestion
                anchor.download = 'assetExport';
```

**Review by bossenti — COMMENTED** [review]
Just did a quick test and it works as expected 🚀

File: ui/src/app/assets/components/asset-overview/asset-overview.component.ts (hunk 2 above)
Comment (IsaakKrut) [reply ui/src/app/assets/components/asset-overview/asset-overview.component.ts]: renamed the filename

File: ui/src/app/assets/components/asset-overview/asset-overview.component.ts (hunk 1 above)
Comment (IsaakKrut) [reply ui/src/app/assets/components/asset-overview/asset-overview.component.ts:164]: Updated to use file-saver

**Review by bossenti — APPROVED** [review]

**Pull request merged**